src/main/java/net/woody/chess/Representation.java

```diff
@@ -0,0 +1,17 @@
+package net.woody.chess;
+
+public enum Representation {
+
+    PAWN('p'), KNGINT('n'), ROOK('r'), BISHOP('b'), QUEEN('q'), KING('k');
```

Comment:
```suggestion
    PAWN('p'),
    KNGINT('n'),
    ROOK('r'),
    BISHOP('b'),
    QUEEN('q'),
    KING('k');
```

Enum에 속성이 있을 때는 요렇게 써주세요~

Reply:
Enum에 속성 정보가 char 하나라서 신경써주지 않았는데 들켰네요..!👀 수정했습니다!😁